lib/davinci_pas_test_kit/generated/v2.0.1/server_suite.rb

```diff
@@ -12,158 +12,7 @@ class ServerSuite < Inferno::TestSuite
-        > [Da Vinci External Reference page](https://confluence.hl7.org/display/DVP/Da+Vinci+Reference+to+External+Standards+and+Terminologies) 
-        > for details on how to get this mapping.
-        >
-        > There are many situationally required fields that are specified in the X12 TRN03 guide that do not have guidance
-        > in this Implementation Guide. Implementers need to consult the X12 PAS guides to know the requirements for these
-        > fields.
-        >
-        > Several of the profiles will require use of terminologies that are part of X12 which we anticipate being made
-        > publicly available. At such time as this occurs, the implementation guide will be updated to bind to these as
-        > external terminologies.
-
-        The implications of this reliance on proprietary information that is not publicly available means that this test
-        kit:
-
-        - Cannot verify the correct usage of X12-based terminology: terminology requirements for all elements bound to X12
-          value sets will not be validated.
-        - Cannot verify the meaning of codes: validation that a given response conveys something specific, e.g., approval
-          or pending, is not performed.
-        - Cannot verify matching semantics on inquiries: no checking of the identity of the ClaimResponse returned for an
-          inquiry, e.g., that it matches the input or the original request.
-
-        These limitations may be removed in future versions of these tests. In the meantime, testers should consider these
-        requirements to be verified through attestation and should not represent their systems to have passed these tests
-        if these requirements are not met.
-
-        ### Underspecified Subscription Details
-
-        The current PAS specification around subscriptions and notifications
-        is not detailed enough to support testing. Notably, 
-        - There are no examples of what a notification payload looks like
-        - There is no instruction on how to turn the notification payload into an inquiry
-
-        Once these details have been clarified, validation of the notification workflows
-        will be added to these tests.
-
-        ### Future Details
-
-        The PAS IG places additional requirements on servers that are not currently tested by this test kit, including
-
-        - Prior Authorization update workflows
-        - Requests for additional information handled through the CDex framework
-        - PDF, CDA, and JPG attachments
-        - US Core profile support for supporting information
-        - Inquiry matching and subsetting logic
-        - Inquiry requests from non-submitting systems
-        - Collection of metrics
-
-        These and any other requirements found in the PAS IG may be tested in future versions of these tests.
-      )
+      description File.read(File.join(__dir__, '..', '..', 'docs', 'server_suite_description_v201.md'))
```

Comment:
In order for this to work when this is shipped elsewhere via a gem, you need to add to gemspec.  I'll push that up real quick.

Reply:
thanks for catching